.travis.yml

```diff
@@ -6,14 +6,18 @@ services:
 cache:
   directories:
   - "$HOME/.cache/pip"
+addons:
+  apt:
+    packages:
```

Comment:
I don't think you need it here: most likely you will bundle the prebuilt bigartm binary with tests or not use it at all